Day-I-Social-Card/social-card/src/components/HeaderComponents/HeaderContainer.js

```diff
@@ -1,2 +1,23 @@
 import React from 'react';
 import './Header.css';
+import HeaderContent from './HeaderContent';
+import ImageThumbnail from './ImageThumbnail';
+const App = () => {
+    return (
+      <div>
+        <HeaderContent />
+        <p>
+          Begin by exploring the `components` directory. You'll notice we have a
+          few files that we've already included in there to get you started right
+          away building components. You'll need to make sure you include your
+          components that you build in this file to watch your app come to life
+        </p>
+        <p>
+          <strong>
```

Comment:
I don't see too much headway here on the social card. How much time was put into this project yesterday?